common/utils.js

```diff
@@ -2,12 +2,27 @@
 
 import {numpy} from './libs/numpy.js';
 
-function sizeOfShape(shape) {
+export function sizeOfShape(shape) {
   return shape.reduce((a, b) => {
     return a * b;
   });
 }
 
+// This function can read buffer from a http url or a local file path.
```

Comment:
Suggested annotation: 

"This function is used for reading buffer from a given url, which will be exported to node.js environment as well, so we use 'fs' module for examples ran in node.js and `fetch()` method for examples ran in browser".